### ISSUE TITLE
Icons don't display using ls (iTerm2 on Mac OS X 10.12)

### ISSUE THREAD

**Issue by heyitsols** [illinoisjackson/even-better-ls#18]
Hello

I am using Mac OS X 10.12 and iTerm2 as my terminal emulator

Running `ls_colors_generator` shows the icons as expected, however running the patched `ls` binary doesn't seem to respect the LS_COLORS variable

The patched `ls` output is different from the system `ls`, so I know that this is the binary being used. For clarity, here is an md5 of the file if comparison is needed, not sure what other information you would need to assist with this

```
 in ~ md5 /Users/ole09/bin/lsi
MD5 (/Users/ole09/bin/lsi) = 6228598ea957078cc985ffb684cc5e9f
```

Thanks,

Oliver

**Comment by mnurzia** [comment]
Are you sure that you have nerd-fonts installed? You can get a compatible font [here.](http://www.github.com/ryanoasis/nerd-fonts)

**Comment by heyitsols** [comment]
Yep, the fonts are definitely there, as the `ls_colors_generator` displays them fine

![screen shot 2017-06-28 at 11 20 32](https://user-images.githubusercontent.com/5894977/27632549-e2efa69e-5bf3-11e7-8f67-9ecd1c6c78cd.png)

**edit**

Even the colours don't work
```
LS_COLORS=$(ls_colors_generator)
run_ls() {
  lsi --color=auto -w $(tput cols) "$@"
}
alias ls="run_ls"
```
![screen shot 2017-06-28 at 12 15 47](https://user-images.githubusercontent.com/5894977/27634546-89b276ee-5bfb-11e7-90d5-0fc32b0c7dc7.png)

**Comment by heyitsols** [comment]
Found it, the LS_COLORS needed exporting too

Would you be averse to a PR that will set some variables and additional commands if the host is a Mac?

**Comment by heyitsols** [comment]
#19 Add additional commands if install target is macOS

**Comment by avshyz** [comment]
bump?

**Comment by Noammac** [comment]
Dev is dead, get a fork

**Comment by chrisvacc** [comment]
@muggahtee, does your commit work with Mojave?

I’ve never cloned a commit before but this isn’t working:

```
git clone https://github.com/mnurzia/even-better-ls.git -b mac_os_script_changes ~/even-better-ls-mac
Cloning into '/Users/chrisvaccaro/even-better-ls-mac'...
```
```
fatal: Remote branch mac_os_script_changes not found in upstream origin
```


Or is it just better to clone `ev0rtex/even-better-ls`

**Comment by heyitsols** [comment]
This project is dead it seems. But the above branch only exists on my fork (so replace mnurzia with muggahtee)

However, I’d recommend [LSD](https://github.com/Peltoche/lsd) which is similar but maintained
